src/xdocs/checks/design/hideutilityclassconstructor.xml.template

```diff
@@ -57,6 +66,21 @@ public class StringUtils // not final to allow subclassing
                    value="resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java"/>
             <param name="type" value="code"/>
         </macro>
+
+        <p id="Example2-config">
+          To configure the check:
```

Comment:
This verbiage needs to change to specify what kind of configuration we are doing here. This is the exact same verbiage as Example 1, so if someone sees these 2 they will question why didn't the first example already show how to configure and what makes Example 2 any different. Look at our other documentation examples.

Reply:
done

Reply:
I don't see how this is done. Example 2 still just says `To configure the check`. I am expecting something like `To configure the check to ...`.